packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -723,25 +744,38 @@ async function handler(
 
   const bookingLocation = getLocationValueForDB(locationBodyString, eventType.locations);
   const customInputs = getCustomInputsResponses(reqBody, eventType.customInputs);
-  const teamMemberPromises =
-    users.length > 1
-      ? users.slice(1).map(async function (user) {
-          return {
-            email: user.email || "",
-            name: user.name || "",
-            timeZone: user.timeZone,
-            language: {
-              translate: await getTranslation(user.locale ?? "en", "common"),
-              locale: user.locale ?? "en",
-            },
-          };
-        })
-      : [];
+
+  const teamDestinationCalendar: DestinationCalendar[] = [];
+
+  // Organizer or user owner of this event type it's not listed as a team member.
+  const teamMemberPromises = users.slice(1).map(async (user) => {
```

Comment:
This was working good just made some little improvements, but teamMemberPromises shouldn't add organiser here.

Reply:
Probably we can rename this const better, so it indicates that all team members should be here except host[0].